editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -44,6 +44,9 @@ Node2D *Polygon2DEditor::_get_node() const {
 void Polygon2DEditor::_set_node(Node *p_polygon) {
 
 	node = Object::cast_to<Polygon2D>(p_polygon);
+	if (node) {
+		splits_prev = node->get_splits();
```

Comment:
split_prev is only used in the input functions, so i'm not quite sure why it needs to go there. It's not used for drawing either.

Reply:
I did at `if (uv_move_current == UV_MODE_REMOVE_SPLIT)` at first. 
https://github.com/volzhs/godot/blob/dfb6102d19c75c00fbdde1fe956ab7a51eacae91/editor/plugins/polygon_2d_editor_plugin.cpp#L567

but I found that `split_prev` is used in many places.
is it better to use in that specific `if UV_MODE_REMOVE_SPLIT`?
it works in my test too.